pkgs/tools/video/xjadeo/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, stdenv, autoreconfHook, fetchFromGitHub, ffmpeg, freetype, libGLU
+, libjack2, liblo, libX11, libXv, pkg-config, portmidi, xorg }:
+stdenv.mkDerivation {
+  pname = "xjadeo";
+  version = "0.8.10";
+
+  src = fetchFromGitHub {
+    owner = "x42";
+    repo = "xjadeo";
+    rev = "c56548cad7b9bd358eb2c776de5ec5323f41006e";
+    sha256 = "0cwh3zimrysyank3x1bcysdz59f8srqdca7jwv97ri3y63dlhifj";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    ffmpeg
+    libjack2
+    libX11
+    pkg-config
+    xorg.libXext
+    xorg.libXpm
+
+    # These are recommended in the README, but not seemingly unnecessary for a
+    # successful build? Possibly required at runtime in some cases? Should
+    # consider investigating and splitting these into options in the future.
+    freetype
+    libGLU
+    liblo
+    libXv
+    portmidi
+  ];
+
+  meta = with lib; {
+    description = "The X Jack Video Monitor";
+    longDescription = ''
+      Xjadeo is a software video player that displays a video-clip in sync with
+      an external time source (MTC, LTC, JACK-transport). Xjadeo is useful in
+      soundtrack composition, video monitoring or any task that requires to
+      synchronizing movie frames with external events.
+    '';
+    homepage = "http://xjadeo.sf.net";
+    license = licenses.gpl2;
```

Comment:
gpl2Only or gpl2Plus?

Reply:
I'm unsure which is which to be honest!

Here's the xjadeo license:

https://github.com/x42/xjadeo/blob/master/COPYING

and here's the harvid one:

https://github.com/x42/harvid/blob/master/COPYING

any advice on which to use and I'll make the change.

Ardour currently has `gpl2`, and seeing as it is by the same group of authors and those COPYING pages were also GPL2 I thought I'd copy suit, but I understand it's better to be more specific when possible.

Reply:
It is plus https://github.com/x42/xjadeo/blob/master/src/xjadeo/main.c#L8